app/helpers/application_helper.rb

```diff
@@ -28,6 +32,18 @@ def urlify(electronic_access, document)
     end
   end
 
+  def umlaut_fulltext_eligible?(document)
```

Comment:
Rather than copy the same method twice, could you include `Blacklight::Solr::Document::Marc` in the helper? Then you could change line 21/25 back to what it was before.